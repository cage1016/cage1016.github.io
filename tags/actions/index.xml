<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>actions - Tag - KaiChu</title><link>https://kaichu.io/tags/actions/</link><description>actions - Tag - KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Fri, 07 May 2021 08:39:11 +0000</lastBuildDate><atom:link href="https://kaichu.io/tags/actions/" rel="self" type="application/rss+xml"/><item><title>Github Script Repos and Collaborators List Sync</title><link>https://kaichu.io/posts/github-script-repos-and-collaborators-list-sync/</link><pubDate>Fri, 07 May 2021 08:39:11 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/github-script-repos-and-collaborators-list-sync/</guid><description>記錄了如何將整理 Github 公用帳號列表及成員的簡單服務從 GAE cron job 搬到 Github Actions 的使用方式</description></item></channel></rss>