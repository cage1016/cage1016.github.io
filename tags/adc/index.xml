<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>ADC on KaiChu</title><link>https://kaichu.io/tags/adc/</link><description>Recent content in ADC on KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><lastBuildDate>Tue, 21 Jul 2015 10:04:40 +0800</lastBuildDate><atom:link href="https://kaichu.io/tags/adc/index.xml" rel="self" type="application/rss+xml"/><item><title>Google Announce Application Default Credentials (ADC)</title><link>https://kaichu.io/posts/adc/</link><pubDate>Tue, 21 Jul 2015 10:04:40 +0800</pubDate><guid>https://kaichu.io/posts/adc/</guid><description>&lt;p>今天 Google 公佈了 &lt;a href="http://googlecloudplatform.blogspot.tw/2015/07/Easier-Auth-for-Google-Cloud-APIs-Introducing-the-Application-Default-Credentials-feature.html">Application Default Credentials (ADC)&lt;/a>，一個可以讓使用者更方便在 GCP 上去界接其他的需要使用 OAuth 存取的服務如 &lt;a href="https://cloud-dot-devsite.googleplex.com/storage">Google Cloud Storage&lt;/a>、&lt;a href="https://cloud-dot-devsite.googleplex.com/bigquery">Google BigQuery&lt;/a>。這對常常寫 GAE 的我來說又更方便了。&lt;/p>
&lt;p>在 GCP 專案建立之後，預設會自動產生 &lt;a href="https://developers.google.com/accounts/docs/OAuth2ServiceAccount">Service Accounts&lt;/a>，
這些內建的 Service Accounts 在進行 Server to Server 的存取時只需要應用程式本身的認証，直接使用 &lt;code>AppAssertionCredentials&lt;/code> 可以不需透過 &lt;code>Flow&lt;/code> 來建立 &lt;code>Credentials&lt;/code>物件&lt;/p></description></item></channel></rss>