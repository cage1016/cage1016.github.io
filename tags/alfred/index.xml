<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>alfred - Tag - KaiChu</title><link>https://kaichu.io/tags/alfred/</link><description>alfred - Tag - KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Thu, 08 Dec 2022 14:29:41 +0000</lastBuildDate><atom:link href="https://kaichu.io/tags/alfred/" rel="self" type="application/rss+xml"/><item><title>Golang Base Alfred Code Sign and Notarize</title><link>https://kaichu.io/posts/golang-base-alfred-code-sign-and-notarize/</link><pubDate>Thu, 08 Dec 2022 14:29:41 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/golang-base-alfred-code-sign-and-notarize/</guid><description>After solving the problem with Gon Sign ErrSecInternalComponent in the previous post, we can now use Gon to sign and notarize code in Github Action to create a complete release process.</description></item><item><title>Gon Sign ErrSecInternalComponent</title><link>https://kaichu.io/posts/gon-sign-errsecinternalcomponent/</link><pubDate>Tue, 15 Nov 2022 16:12:38 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/gon-sign-errsecinternalcomponent/</guid><description>Gon is a tool developed by Hashicorp founder Mitchell Hashimoto that makes the whole process of code signing easy. This article documents the problems I encountered with Gon code sign and how I solved them.</description></item><item><title>Alfred Open Chinese Convert</title><link>https://kaichu.io/posts/alfred-open-chinese-convert/</link><pubDate>Sun, 10 Jul 2022 14:42:06 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/alfred-open-chinese-convert/</guid><description>Since Mac OS finally removed support for Python2 in 12.3, the workflows written in Python2 in Alfred 4 are no longer available. The amowu/alfred-chinese-converter I was using before is also no longer updated (it's still in Alfred 2). So I built one with Golang + awgo + opencc, eliminating the dependency on Python, and Alfred upgrades no longer run into Python version problems.</description></item><item><title>Alfred Change Case</title><link>https://kaichu.io/posts/alfred-change-case/</link><pubDate>Tue, 07 Dec 2021 01:32:42 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/alfred-change-case/</guid><description>Alfred 是一個在 Mac 上面增加生產力的工具，舉凡工程師常用的 Github、Google Drive 搜尋到 LeetCode Search 等原都可以在 Alfred 完成，簡直就是太方便了。自己也有曾經想要開發自己的 Alfred 應用，直到遇到 deanishe/awgo 之後就像撿到寶一樣，作為一個 Gopher 也找到適合的工具來開發自己的 Alfred 應用。</description></item></channel></rss>