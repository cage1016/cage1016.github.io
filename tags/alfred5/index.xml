<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>alfred5 - Tag - KaiChu</title><link>https://kaichu.io/tags/alfred5/</link><description>alfred5 - Tag - KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Sun, 03 Sep 2023 08:09:25 +0000</lastBuildDate><atom:link href="https://kaichu.io/tags/alfred5/" rel="self" type="application/rss+xml"/><item><title>Golang Alfred Workflow Generator Ak</title><link>https://kaichu.io/posts/golang-alfred-workflow-generator-ak/</link><pubDate>Sun, 03 Sep 2023 08:09:25 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/golang-alfred-workflow-generator-ak/</guid><description>A generator for golang alfred workflow that helps you create boilerplate code.</description></item></channel></rss>