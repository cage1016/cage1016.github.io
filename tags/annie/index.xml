<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>annie on KaiChu</title><link>https://kaichu.io/tags/annie/</link><description>Recent content in annie on KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><lastBuildDate>Sat, 20 Jun 2020 12:13:48 +0000</lastBuildDate><atom:link href="https://kaichu.io/tags/annie/index.xml" rel="self" type="application/rss+xml"/><item><title>iOS ringtone maker from YouTube video</title><link>https://kaichu.io/posts/ios-ringtone-maker-from-youtube/</link><pubDate>Sat, 20 Jun 2020 12:13:48 +0000</pubDate><guid>https://kaichu.io/posts/ios-ringtone-maker-from-youtube/</guid><description>&lt;p>最近大竹弟打電話問說 iOS 鈴聲在 Garageband 上面製作的檔案怎麼這麼大，其實只有 5 秒的鈴聲&lt;/p>
&lt;p>先來說一下 iPhone 上配置客製的鈴聲有二種方式&lt;/p>
&lt;ul>
&lt;li>iTunes&lt;/li>
&lt;li>Garageband 上編輯直接輸出至鈴聲&lt;/li>
&lt;/ul>
&lt;p>由於 macOS 10.15 版本就沒有 itunes 可以用，所以基本上網路的教學大也是直接在 Garageband 上直接編輯後直接輸出至 iPhone 的鈴聲&lt;/p>
&lt;p>以前我自己也是換過不少鈴聲，以前是透過 &lt;a href="https://sur.ly/o/audiko.tw/AA000014">Free Ringtones for Android and iPhone. Free Ringtone Maker - Audiko&lt;/a> 選擇喜歡的檔案下載安裝&lt;/p>
&lt;p>雖然現成的工具方便，不過有時候找不到自己喜歡的歌曲，那就土砲一個吧&lt;/p></description></item></channel></rss>