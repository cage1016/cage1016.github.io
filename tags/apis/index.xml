<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>APIs - Tag - KaiChu</title><link>https://kaichu.io/tags/apis/</link><description>APIs - Tag - KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Tue, 21 Jul 2015 10:04:40 +0800</lastBuildDate><atom:link href="https://kaichu.io/tags/apis/" rel="self" type="application/rss+xml"/><item><title>Google Announce Application Default Credentials (ADC)</title><link>https://kaichu.io/posts/adc/</link><pubDate>Tue, 21 Jul 2015 10:04:40 +0800</pubDate><author>Author</author><guid>https://kaichu.io/posts/adc/</guid><description>今天 Google 公佈了 Application Default Credentials (ADC), 一個可以讓使用者更方便在 GCP 上去界接其他的需要使用 OAuth 存取的服務</description></item></channel></rss>