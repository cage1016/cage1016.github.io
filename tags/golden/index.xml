<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>golden - Tag - KaiChu</title><link>https://kaichu.io/tags/golden/</link><description>golden - Tag - KaiChu</description><generator>Hugo -- gohugo.io</generator><language>en</language><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Wed, 06 Sep 2023 15:29:23 +0000</lastBuildDate><atom:link href="https://kaichu.io/tags/golden/" rel="self" type="application/rss+xml"/><item><title>Alfred Theme Black Golden</title><link>https://kaichu.io/posts/alfred-theme-black-golden/</link><pubDate>Wed, 06 Sep 2023 15:29:23 +0000</pubDate><author>Author</author><guid>https://kaichu.io/posts/alfred-theme-black-golden/</guid><description>Alfred Theme Black Golden</description></item></channel></rss>